src/lib/import-export/import/importers/importer.ts

```diff
@@ -96,24 +96,25 @@ abstract class BaseImporter extends EventEmitter implements Importer {
 		}
 
 		const studioUrl = `http://localhost:${ server.details.port }`;
+		const oldUrl = currentSiteUrl.trim();
+		const oldUrlHttps = oldUrl.replace( 'http://', 'https://' );
+		const oldUrlHttp = oldUrl.replace( 'https://', 'http://' );
+
+		for ( const urlToReplace of [ oldUrlHttps, oldUrlHttp ] ) {
+			const { stderr, exitCode } = await server.executeWpCliCommand(
+				`search-replace '${ urlToReplace }' '${ studioUrl }' wp_posts wp_postmeta wp_options`,
```

Comment:
I am running it on the `wp_options` table to cover for the site url that we replaces in `option get siteurl` and two other tables that are common for images.

Alternatively, we can run it on the whole site. Happy to change this.